src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceActivity.cs

```diff
@@ -109,13 +109,13 @@ public partial class DiagnosticListener
     {
         public override void OnActivityImport(Activity activity, object payload)
         {
```

Comment:
It seems like if `Activity == null` this can nullref, but can the payload be null?

Reply:
Seems could be null will check again